src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,126 @@
+import classNames from 'classnames';
+import React, { useCallback, useState } from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  removeTodo: (id: number) => Promise<void>;
+  loadingTodoId: number[];
+  loading?: boolean;
+  updateTodo: (id: number, titleOrCompleted: string | boolean) => Promise<void>;
```

Comment:
The parameter name 'titleOrCompleted' can be confusing. It's better to use separate parameters for 'title' and 'completed' to make the code more readable.